ts-obs/cmd/helmInstall.go

```diff
@@ -0,0 +1,83 @@
+package cmd
+
+import (
+    "errors"
+	"fmt"
+    "io"
+    "os"
+    "os/exec"
+
+	"github.com/spf13/cobra"
+)
+
+// helmInstallCmd represents the helm install command
+var helmInstallCmd = &cobra.Command{
+	Use:   "install",
+	Short: "Installs Timescale Observability",
+	RunE: helmInstall,
+}
+
+func init() {
+	helmCmd.AddCommand(helmInstallCmd)
+    helmInstallCmd.Flags().StringP("name", "n", "ts-obs", "Release name")
```

Comment:
If we want the option for people to install two or more Helm charts, then release name can't be set to default, or there's going to be a conflict. Or we can maybe automatically check if there is already an instance of the chart installed. The global flag solution seems like it would be annoying to type every time. ENV variables could work but then they would need to be manually changed every time you want to do operations in a different set of pods.